lua/wikis/commons/MatchGroup/Legacy.lua

```diff
@@ -323,14 +323,6 @@ function MatchGroupLegacy:handleDetails(isReset, match1params, match)
 	Table.deepMergeInto(match, details)
 end
 
----@param isReset boolean
----@param match table
-function MatchGroupLegacy:handleFinished(isReset, match)
-	if isReset then return end
```

Comment:
What was the reason reset matches were never set to finished?

Reply:
no idea
but it does no harm